src/Huntress/Plugin/Remind.php

```diff
@@ -143,8 +152,9 @@ public static function remindMe(EventData $data): ?Promise
             $tzinfo = sprintf("%s (%s)", $time->getTimezone()->toRegionName(), $time->getTimezone()->toOffsetName());
             $embed->addField("Detected Time",
                 $time->toDayDateTimeString() . PHP_EOL . $tzinfo . PHP_EOL . $time->longRelativeToNowDiffForHumans(2));
+            $embed->setFooter(sprintf("Delete with `!rem del %s`", Snowflake::format($snow)));
```

Comment:
minor gripe, but if we're using the short command `!rem` here, we should change the help text to match. keira ocd brain goes brrr

Reply:
✔️